platform/linux-generic/odp_packet_io.c

```diff
@@ -1445,9 +1445,12 @@ int odp_pktio_capability(odp_pktio_t pktio, odp_pktio_capability_t *capa)
 	else
 		ret = single_capability(capa);
 
-	/* The same parser is used for all pktios */
-	if (ret == 0)
+	if (ret == 0) {
+		/* The same parser is used for all pktios */
 		capa->config.parser.layer = ODP_PROTO_LAYER_ALL;
+		/* Header skip is supported */
+		capa->set_op.op.skip_offset = 1;
```

Comment:
It seems that classifier does not actually use skip offset when processing packets. It is set, but not used.

It would be good to implement it. Otherwise, we need to set it "unsupported" here.

Reply:
Ok. I didn't look at implementation in detail and thought it is supported as the validation test was added. 
I'll mark it as not supported.

Reply:
Also, current API spec for skip offset seems not be complete ("Number of bytes the classifier must skip"). To me it should be documented that this setting affect also parser (all input side offloads in general ?). I think the use case of this offset is to ODP packet input HW to ignore any proprietary headers (e.g. used by backplane switching) before Ethernet frame start.

Reply:
I agree. Will update the text as above.